spec/src/main/asciidoc/rest-endpoints.adoc

```diff
@@ -168,6 +168,207 @@ myClass_myMethod_seconds_max{mp_scope="vendor"} 0.05507899 <3>
 
 ^2^ This will return an array of `PercentileValue` objects that will need to be iterated through to find the specific percentile and its value
 
+=== Additional output and customization for histograms and timers
```

Comment:
this doesn't seem like it belongs in the rest-endpoints doc, since it's applicable to all histograms and timers (not just those used for REST).

Reply:
Moved to it's own adoc.